edx_sga/sga.py

```diff
@@ -269,8 +268,11 @@ def upload_assignment(self, request, suffix=''):
         student_item_dict = self.get_student_item_dict()
         submissions_api.create_submission(student_item_dict, answer)
         path = self.file_storage_path(sha1, upload.file.name)
-        if not default_storage.exists(path):
-            default_storage.save(path, File(upload.file))
+        log.info("Saving file: %s at path: %s for user: %s", upload.file.name, path, user.username)
+        if default_storage.exists(path):
+            # save latest submission
+            default_storage.delete(path)
```

Comment:
Shouldn't the file not be there at all? When would it already exist?

Reply:
yes, this is for 2nd time upload, user can upload as mush as he want before finalizing assignment.